ios/Classes/SwiftKlippaIdentityVerificationSdkPlugin.swift

```diff
@@ -150,4 +160,9 @@ public class SwiftKlippaIdentityVerificationSdkPlugin: NSObject, FlutterPlugin,
         resultHandler!(FlutterError.init(code: E_CANCELED, message: "The user canceled", details: nil));
         resultHandler = nil;
     }
+
+    public func contactSupportPressed() {
+        resultHandler!(FlutterError.init(code: E_CANCELED, message: "The user pressed contact support", details: nil));
```

Comment:
@RobinFarmer We probably need another code for this? How else would the developer recognize this "canceled" method?

Reply:
Adjusted it to `E_SUPPORT_PRESSED`.